packages/vue-i18n-routing/src/compatibles/utils.ts

```diff
@@ -58,6 +58,21 @@ function split(str: string, index: number) {
   return result
 }
 
+export function routeToObject(route: Route | RouteLocationNormalizedLoaded) {
```

Comment:
Below is your Description on this PR:

> Does not work for @nuxtjs/i18n or maybe it is a Nuxt related issue, I noticed that the route object was a proxy and accessing each property did work. So I added a function to destructure each property by name instead of using a spread operator.
> Unfortunately I don't know why the behaviour is like this, and I don't think I can add tests for this case as it may be Nuxt related.

Let’s write the comment !, so maintainer can understand the background when we read this code.

Reply:
I have added a short description now, hopefully it is informative enough.